lms/djangoapps/discussion_api/tests/utils.py

```diff
@@ -392,3 +400,55 @@ def make_paginated_api_response(results=None, count=0, num_pages=0, next_link=No
         },
         "results": results or []
     }
+
+
+class ProfileImageMixin(object):
```

Comment:
We need to make this less dependent on other mixins/test case. Make it sure it can stand alone without any hidden properties like user, storage and assertion statements.